apps/server/lib/http/auth.ts

```diff
@@ -0,0 +1,70 @@
+import {
+	JellyfishKernel,
+	SessionContract,
+} from '@balena/jellyfish-types/build/core';
+import bcrypt from 'bcrypt';
+import _ from 'lodash';
+
+// TODO: Move this definition elsewhere, and inject it into the middleware.
+const validateSessionIdAndTokenFunc =
+	(context, jellyfish: JellyfishKernel) =>
+	async (session: string, sessionToken?: string): Promise<boolean> => {
+		return jellyfish
+			.getCardById<SessionContract>(context, jellyfish.sessions!.admin, session)
+			.then((result) => {
```

Comment:
I usually prefer to go with `await` when possible instead of `then` to make things a bit clearer. It's not too bad here since its only one level deep, but I'd still prefer to use `await` instead to not only match the rest of the code base, but also to make future edits that may require their own `await`/`then` easier.

Reply:
Yes, it makes sense. Replaced this with `await` as suggested.